parts/clinical-use-cases.md

```diff
@@ -0,0 +1,93 @@
+  - "like looking for a needle in a haystack ... of needels"
+  - Huge amounts of data, 99.9% no one really knows what to do with or how to store and backup
+    - It's a little like nuclear power plants - we generate a lot of "waste" that no ones has figured out a plan to do with
+
+## Why are we doing this? Non-python
+- In the beginning there was ... Excel
+  - Thousands of rows in an unwieldy, and inherently static grid
+
+- Reasons to improve
+  - we are dealing with routine tasks more than exploratory research
+    - prioritize variants and focus less on filtering for example
+  - quicker turn-around times
+  - automate parts that can be automated (free up doctors)
+
+- We want to revolutionize healthcare.
+  - [insert Clinical Genomics objectives]
+
+If we do it right it can help both doctors and patients. If we don't the whole project will be slowed down.
+
+## What are we going to cover?
+- Flask app + MongoDB/MongoEngine
+- Logging??
+
+Hopefully I can share some best practises around working with Flask and teach you something new to test in your own projects.
+
+## How does the field look like? Current situation (not essential)
+- Lots of activity around the world (GB)
+- Many "competitors" focus on research + more complicated to work with
+
+## What is the unique problems? - how are you solving them?
+- close collaboration with end users
+  - blessing and a curse
+  - likely to deliver what they want
+  - risk of building something that is too tailored to be useful for others
+
+- clinical/sensitive nature of data
+  - strict guidelines
+  - simple or low hanging fruit features are complicated
+  - e.g. personally idetifiable data; it's in the nature of the data!
+    - can't share anything between different entetites (customer clinics)
+    - global frequencies would greatly benefit the end product
+
+### Server side rendering galore
+- in the beginning there was ... chaos
+  - Flask+Tornado + lots of JavaScript
+    - mostly non-updating data, big tables, sluggish performance
+  - we repented: rely on server side + minimize client side
+  - easier for us to collaborate
+  - it more performant
+  - iteration and testing *much* simpler
+  - users just want to get their work done
+  - doing it in Python >> JavaScript
+
+- tip: look at GitHub site and copy conventions!
+  + How they order URLs
+  + What's server side vs. client side rendered
+  + Log out issues resolved when adding a form
+
+- leverage components (Vue.js and SCSS) on what client side we have
+  - don't need to mess too much with jQuery.
+  - Jinja filter for {{}}
+  - Web components syntax
+
+### Why did we opt for MongoDB?
+- Bring up simplified example for comments
+  - This is different than most regular setups because we need to separates comments between institutes and cases...
+  - Much more intuitive in MongoDB
+
+To also get into some actual Python code and conventions I could talk about using MongoEngine as an interface ^^ to MongoDB.
```

Comment:
I think that if you're going to talk about MongoDB, you should do it through MongoEngine, because that's the Python interface, and this is a Python conference :-)